packages/components/modules/profiles/Members/MembersList/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useMemo } from 'react'
+
+import { LoadingState as DefaultLoadingState } from '@baseapp-frontend/design-system'
+
+import { Box, Divider, Typography } from '@mui/material'
+import { usePaginationFragment } from 'react-relay'
+import { Virtuoso } from 'react-virtuoso'
+
+import { UserMembersListFragment } from '../../graphql/queries/UserMembersList'
+import DefaultMemberItem from '../MemberItem'
+import { MemberStatuses, NUMBER_OF_MEMBERS_TO_LOAD_NEXT } from '../constants'
+import { MemberListProps } from '../types'
+
+const MembersList: FC<MemberListProps> = ({
+  meRef,
+  MemberItem = DefaultMemberItem,
+  LoadingState = DefaultLoadingState,
+  LoadingStateProps = {},
+  membersContainerHeight = 400,
+}) => {
+  const { data, loadNext, hasNext, isLoadingNext } = usePaginationFragment(
+    UserMembersListFragment,
+    meRef,
+  )
+
+  const members = useMemo(
+    () =>
+      data?.profile?.members?.edges.filter((edge) => edge?.node).map((edge) => edge?.node) || [],
+    [data?.profile?.members?.edges],
+  )
+
+  const renderLoadingState = () => {
+    if (!isLoadingNext) return <Box sx={{ paddingTop: 3 }} />
+
+    return (
+      <LoadingState
+        sx={{ paddingTop: 3, paddingBottom: 1 }}
+        CircularProgressProps={{ size: 15 }}
+        {...LoadingStateProps}
+      />
+    )
```

Comment:
Instead of using paddingTop and paddingBottom, you can use, respectively, pt and pb. This alisases, inside of the sx props, make the code cleaner.